### DISCUSSION TITLE
Is multiple DICOM sequence synchronization supported?

### DISCUSSION THREAD

**Discussion by SleLaugh** [nroduit/Weasis#459] [Q&A]
Hi @nroduit , Thank you so much for creating this kind of framework.
What I want to know is does Weasis support opening multiple sequences and synchronizing them?

Specifically, when I set the sync sequence mode to TILE, even if I open a multi-form layout such as 2x3, it will only display the image of one sequence, and the mouse scroll operation will only operate the DICOM of this sequence.

Whether it is supported to display multiple sequences when the synchronization mode is set to TILE, and to synchronize these open sequences when the mouse is scrolling.

**Comment by nroduit** [comment]
Tile mode can only display a single series, and spreads images across all available views. If there were more than one series, we'd have to delimit the spread over the views. I don't think I've seen a synchronization mode between several tiled series in other software. Moreover, I'm not sure what rules apply.

Weasis has chosen not to combine a viewport that contains multiviews in Tile mode, in order to keep things simple for users.

However, it is possible to open a second tab (right-click on thumbnail) and to display both tabs at the same time (either on a second screen or by drag-drop to share the main space). After that, you can display two series (or the same one) in Tile mode but without synchronization.

[Stack synchronization between several tabs](https://github.com/nroduit/Weasis/issues/433) is now supported by version 4.2.0.

**Reply by SleLaugh** [reply]
I tried it, but it didn't always work.

Here's what I did: I opened two sequences of the same patient on two tabs.
And change the synchronization mode of both tabs to TILE, then select a TAB and scroll with the mouse. 
However, at this time, only the selected TAB has the scrolling effect, while the other tabs do not change.

I want to know how I can use this function correctly.

And are there any restrictions?
Are only sequences for a single patient allowed, or are sequences for multiple patients supported at the same time?

**Comment by nroduit** [comment]
Synchronization between multiple tabs only works in stack mode (2D views). Currently, tile mode has no synchronization outside its container. And as mentioned above, I haven't seen this in other software.
